app/app/send-modal.tsx

```diff
@@ -68,6 +79,16 @@ export default function SendModal() {
     setIsLoadingTransfer(false);
 
     router.back();
+
+    // Refetch transactions
+    const res = await getPayments(user!.token, {
```

Comment:
I don't remember here why we need the list of transactions. Can you remind me?

Reply:
After making a transaction, we'd need to update the list of the state of the payments to include that transaction. 

It's a background API call, and wouldn't disrupt any other action.

Reply:
Can't you just do a pop and push in place?

Reply:
yeah, just did that ! Thanks for pointing that out

-- I didn't realize the implementation behind the transfer page list display